www/gatsby-config.js

```diff
@@ -62,5 +62,13 @@ module.exports = {
         ],
       },
     },
+    {
+      resolve: `gatsby-plugin-segment-js`,
+      options: {
+        prodKey: process.env && process.env.SEGMENT_KEY || '',
+        devKey: process.env && process.env.SEGMENT_KEY || '',
```

Comment:
I think we can keep these as both the same `process.env.SEGMENT_KEY` as you have it. Not sure the immediate value of having separate variables here.